holidays/countries/cape_verde.py

```diff
@@ -0,0 +1,191 @@
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("Ano Novo"))
+
+        if self._year >= 1991:
+            # Democracy and Freedom Day.
+            self._add_holiday_jan_13(tr("Dia da Liberdade e Democracia"))
+
+        # National Heroes Day.
+        self._add_holiday_jan_20(tr("Dia da Nacionalidade e dos Heróis Nacionais"))
+
+        # Ash Wednesday.
+        self._add_ash_wednesday(tr("Quarta-feira de Cinzas"))
+
+        # Good Friday.
+        self._add_good_friday(tr("Sexta-feira Santa"))
+
+        # Easter Sunday.
+        self._add_easter_sunday(tr("Páscoa"))
+
+        # Worker's Day.
+        self._add_labor_day(tr("Dia do Trabalhador"))
+
+        # International Children's Day.
+        self._add_childrens_day(tr("Dia Mundial da Criança"))
+
+        # Independence Day.
+        self._add_holiday_jul_5(tr("Dia da Independência Nacional"))
+
+        # Assumption Day.
+        self._add_assumption_of_mary_day(tr("Dia da Assunção"))
+
+        # All Saints' Day.
+        self._add_all_saints_day(tr("Dia de Todos os Santos"))
+
+        # Christmas Day.
+        self._add_christmas_day(tr("Natal"))
+
+    def _populate_optional_holidays(self):
+        # Holy Thursday.
+        self._add_holy_thursday(tr("Quinta-Feira Santa"))
+
+        # Mother's Day.
+        self._add_holiday_2nd_sun_of_may(tr("Dia das Mães"))
+
+        # Father's Day.
+        self._add_holiday_3rd_sun_of_jun(tr("Dia dos Pais"))
+
+    def _populate_subdiv_br_public_holidays(self):
+        # Brava Municipal Day.
+        self._add_holiday_jun_24(tr("Dia do Município da Brava"))
+
+    def _populate_subdiv_bv_public_holidays(self):
+        # Municipal Day.
+        self._add_holiday_jul_4(tr("Dia do Município"))
+
+    def _populate_subdiv_ma_public_holidays(self):
+        # Maio Municipal Day.
+        self._add_holiday_sep_8(tr("Dia do Município do Maio"))
+
+    def _populate_subdiv_pr_public_holidays(self):
+        # Praia City Day.
+        self._add_holiday_apr_29(tr("Dia da Cidade da Praia"))
+
+        # Praia Municipal Day.
+        self._add_holiday_may_19(tr("Dia do Município da Praia"))
+
+    def _populate_subdiv_rb_public_holidays(self):
+        # Ribeira Brava Municipal Day.
+        self._add_holiday_dec_6(tr("Dia do Município de Ribeira Brava"))
+
+    def _populate_subdiv_rs_public_holidays(self):
+        # Ribeira Grande de Santiago Municipal Day.
+        self._add_holiday_jan_31(tr("Dia do Município de Ribeira Grande de Santiago"))
+
+    def _populate_subdiv_sl_public_holidays(self):
+        # Municipal Day.
+        self._add_holiday_sep_15(tr("Dia do Município"))
+
+    def _populate_subdiv_sv_public_holidays(self):
+        # São Vicente Municipal Day.
+        self._add_holiday_jan_22(tr("Dia do Município de São Vicente"))
+
+        # Carnival Tuesday.
+        self._add_carnival_tuesday(tr("Terça-feira de Carnaval"))
```

Comment:
I don't found this holiday for São Vicente.

Reply:
It's the ones listed in the main list as `February 13: Shrove Tuesday (regional public holiday)` we discussed above